src/main/java/org/jenkinsci/remoting/engine/RetryUtils.java

```diff
@@ -0,0 +1,93 @@
+package org.jenkinsci.remoting.engine;
+
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.remoting.EngineListenerSplitter;
+import hudson.remoting.Util;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.concurrent.Callable;
+import java.util.function.Function;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.jenkinsci.remoting.util.DurationFormatter;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+@Restricted(NoExternalUse.class)
```

Comment:
the class isn't public so this should be unneeded?

Reply:
you're correct, missed it when applying restrictions.